src/block/icon-label/index.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+// Exit if accessed directly.
```

Comment:
can we rename this file to deprecated.php or something so we can denote that it's for deprecation purposes?